fenix/app/src/main/java/org/mozilla/fenix/settings/logins/fragment/AddLoginFragment.kt

```diff
@@ -80,11 +82,9 @@ class AddLoginFragment : Fragment(R.layout.fragment_add_login), MenuProvider {
         )
 
         initEditableValues()
-
```

Comment:
nits: We should avoid removing these new lines in the final revision of this PR. New lines are used to separate logical blocks in a way so that things are more readable.